compose/utils.py

```diff
@@ -95,4 +95,4 @@ def microseconds_from_time_nano(time_nano):
 
 
 def build_string_dict(source_dict):
-    return dict((k, str(v)) for k, v in source_dict.items())
+    return dict((k, str(v if v else '')) for k, v in source_dict.items())
```

Comment:
Only problem with this is it'll result in the empty string for the integer `0`, which probably isn't what the user expects. `v if v is not None else ''` is probably better. (Would be good to add an arg with a value of `0` to the test, too.)